tools/deviceGen/output/features_members.h

```diff
@@ -0,0 +1,1581 @@
+    /*
+       [REQUIRE] Roadmap 2022 requires support for these, device support is ubiquitous and enablement is unlikely to harm performance
+    */
+    // bool fullDrawIndexUint32 = true;
+    // bool imageCubeArray = true;
+    // bool independentBlend = true;
+    /*
+       I have no clue if these cause overheads from simply being enabled
+    */
+    bool geometryShader = false;
+    bool tessellationShader = false;
+    // bool sampleRateShading = true; // [REQUIRE] ROADMAP 2022 and good device support
+    // bool dualSrcBlend = true; // [REQUIRE] good device support
+    // bool logicOp; // [EXPOSE AS A LIMIT] somewhat legacy features
+    /*
+       [REQUIRE] Roadmap 2022 requires support for these, device support is ubiquitous and enablement is unlikely to harm performance
+    */
+    // bool multiDrawIndirect = true;
+    // bool drawIndirectFirstInstance = true;
+    // bool depthClamp = true;
+    // bool depthBiasClamp = true;
+    // bool fillModeNonSolid = true; // [REQUIRE] good device support
+    bool depthBounds = false; // this is kinda like a weird clip-plane that doesn't count towards clip plane count
+    bool wideLines = false; // good device support, but a somewhat useless feature (constant screenspace width with limits on width)
+    bool largePoints = false; // good device support, but a somewhat useless feature (axis aligned screenspace boxes with limits on size)
+    bool alphaToOne = true; // Some AMD don't support
+    // bool multiViewport = true; // [REQUIRE] good device support
+    // bool samplerAnisotropy = true; // [REQUIRE] Roadmap 2022 requires support for these, device support is ubiquitous
+    /*
+       [DO NOT EXPOSE] these 3 don't make a difference, just shortcut from Querying support from PhysicalDevice
+    */
+    bool textureCompressionETC2;
+    bool textureCompressionASTC_LDR;
+    bool textureCompressionBC;
+    // bool occlusionQueryPrecise = true; // [REQUIRE] ROADMAP 2022 and good device support
+    bool pipelineStatisticsQuery = false;
+    /*
+       [EXPOSE AS LIMIT] Enabled by Default, Moved to Limits
+    */
+    bool vertexPipelineStoresAndAtomics;
+    bool fragmentStoresAndAtomics;
+    bool shaderTessellationAndGeometryPointSize;
+    // bool shaderImageGatherExtended = true; // [REQUIRE] ROADMAP 2024 good device support
+    // bool shaderStorageImageExtendedFormats = true; // [REQUIRE] ROADMAP 2024 good device support
+    bool shaderStorageImageMultisample; // [EXPOSE AS LIMIT] Cannot be always enabled cause Intel ARC is handicapped
+    /*
+       TODO: format feature reporting unimplemented yet for both of the below! (should we move to usage reporting?)
+       [EXPOSE AS LIMIT] always enable, shouldn't cause overhead by just being enabled
+    */
+    bool shaderStorageImageReadWithoutFormat;
+    // bool shaderStorageImageWriteWithoutFormat = true; // [REQUIRE] good device support
```

Comment:
the members that are supposed to be limits make this file hard to review, correct and I'll come back to it